src/components/AvatarMenu/AvatarMenu.styles.ts

```diff
@@ -0,0 +1,51 @@
+import { styled } from '@mui/material/styles';
+import { IconButton, Typography } from '@mui/material';
+import type { PropsStyledAvatarMenu } from './AvatarMenu.types';
+import { theme } from '../../style-library';
+
```

Comment:
This file imports a top-level `theme` from '../../style-library' and then uses it directly inside styled() callbacks. Mixing the imported theme with the styled-provided theme is fragile and breaks theming (and SSR).

**Sugerencia:**
```typescript
Remove the imported theme and use the theme provided by MUI inside the styled callbacks: e.g. styled('div')(({ theme, $fullWidth }) => ({ gap: theme.spacing(1.25) }))
```